### DISCUSSION TITLE
How can I use my own Latex Code for the Title Page to use in PDF, too ?

### DISCUSSION THREAD

**Discussion by paule32** [doxygen/doxygen#11203] [Q&A] [answered]
Hello,
like the Title of this Question says:
How can I use my own Latex Code for the Title Page to use in PDF Documentation ?

**Comment by albert-github** [comment, marked as answer]
Best way would be to modify the file `refman.tex` on the latex directory, here is a.o. `\begin{titlepage}` till `\end{titlepage}` with the code for the generated title page.
